pennylane/interfaces/execution.py

```diff
@@ -598,12 +598,12 @@ def cost_fn(params, x):
 
     if gradient_fn is None:
         # don't unwrap if it's an interface device
-        # if "passthru_interface" in device.capabilities():
-        #     return batch_fn(
-        #         qml.interfaces.cache_execute(
-        #             batch_execute, cache, return_tuple=False, expand_fn=expand_fn
-        #         )(tapes)
-        #     )
+        if "passthru_interface" in device.capabilities() or device.short_name == "default.mixed":

```

Comment:
Why do we have `device.short_name == "default.mixed"` here?

Reply:
The implementation of `default.mixed` is different we don't have multiple device and therefore there is no "passthru_interface" interface, therefore I had to the possibility to do the forward pass here when `gradient_fn` is `None`